File: app/src/main/java/com/robotnec/budget/core/mvp/view/AddAccountView.java
package com.robotnec.budget.core.mvp.view;

import com.robotnec.budget.core.domain.Account;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface AddAccountView extends View {
    void initEditMode(Account account);

    void closeView();
}

File: app/src/main/java/com/robotnec/budget/core/mvp/view/AddAccountView.kt
package com.robotnec.budget.core.mvp.view

import com.robotnec.budget.core.domain.Account

/**
 * @author zak zak@swingpulse.com>
 */
interface AddAccountView : View {
    fun initEditMode(account: Account)

    fun closeView()
}
